### PR TITLE
Update payment docs

### PR DESCRIPTION
**Pull request by vstudenichnik-insoft**

### CODE REVIEW

**Review by greph — APPROVED** [review]

File: content/en/payment.md [hunk 1]
```diff
@@ -304,4 +442,390 @@ await swell.cart.update({
...
+#### Create iDEAL element
+
+Using Stripe elements:
+
+```js
+// Client side
+const stripe = Stripe('pk_test_...');
+​
+const elements = stripe.elements();
+const ideal = elements.create('idealBank', options?);
```
Comment (greph) [inline content/en/payment.md]: @vstudenichnik-insoft is this '?' needed here?
`options?`

Comment (vstudenichnik-insoft) [reply content/en/payment.md]: @greph  `?` means this is an optional parameter

Comment (greph) [reply content/en/payment.md]: @vstudenichnik-insoft , are you just indicating that the parameter is optional in a sudo-code fashion? Wouldn't this throw an error if the user included it in the code?

Comment (greph) [reply content/en/payment.md]: Perhaps we could use a comment instead?
`const ideal = elements.create('idealBank', options); // optional second param`

Comment (vstudenichnik-insoft) [reply content/en/payment.md]: @greph  I think any developer would understand this, but I updated it anyway this way:
`const ideal = elements.create('idealBank', options); // 'options' is an optional object`

Comment (greph) [reply content/en/payment.md]: Thanks @vstudenichnik-insoft ! It's common to copy/paste code so just want to make sure the code is valid.

**Pull request merged**